src/App.tsx

```diff
@@ -17,18 +59,36 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                filterBy={filterBy}
+                handleStatus={handleStatus}
+                query={query}
+                setQuery={setQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading
+                ? (<Loader />)
+                : (
+                  <TodoList
+                    todos={filteredTodos}
+                    selectedTodoId={todoId}
+                    selectTodo={selectTodo}
```

Comment:
Why did you create one more method? was It necessary?
```suggestion
                    selectTodo={setTodoId}
```